INSTALL.md

```diff
@@ -4,25 +4,40 @@
 ### 1. Installing EtherLab
 The proposed development builds upon the [IgH EtherCAT Master](https://etherlab.org/en/ethercat/). Installation steps are summarized here:
 ```shell
+$ sudo apt-get update
+$ sudo apt-get upgrade
+$ sudo apt install git
 $ git clone https://gitlab.com/etherlab.org/ethercat.git
 $ cd ethercat
 $ git checkout stable-1.5
-```
-```shell
-$ ./bootstrap # to create the configure script, if downloaded from the repository
-
-$ ./configure --prefix=/usr/local/etherlab  --disable-8139too --enable-generic # Ethernet driver e1000e not supported for kernels 4.X
+$ sudo apt-get install autoconf
+$ sudo apt-get install libtool
+$ sudo apt-get install pkg-config
+$ sudo apt-get install make
+$ sudo apt-get install build-essential
+$ sudo apt-get install net-tools
+$ sudo rm /usr/bin/ethercat
+$ sudo rm /etc/init.d/ethercat
```

Comment:
is there a practical reason to do this ?

Reply:
yes. If already these files are there then they create issues if we want to reinstall. that's why clearing the ethercat files if already were there for smooth installation.